src/functionalTest/resources/application-integration.yml

```diff
@@ -92,13 +92,13 @@ route:
       sql:INSERT INTO court_venue (epimms_id, site_name, court_name, court_status, court_open_date, region_id, court_type_id, cluster_id, open_for_public,
       court_address, postcode, phone_number, closed_date, court_location_code, dx_address, welsh_site_name, welsh_court_address, venue_name, is_case_management_location,
       is_hearing_location, welsh_venue_name, is_temporary_location, is_nightingale_court, location_type, parent_location, welsh_court_name, uprn,
-      venue_ou_code, mrd_building_location_id, mrd_venue_id, service_url, fact_url, mrd_created_time, mrd_updated_time, mrd_deleted_time, short_court_name, created_time, updated_time)
```

Comment:
looks like these tests are not functional , can we change this directory name to integrationTest